### PR TITLE
Update Data for March in France

### PR DESCRIPTION
**Pull request by mycaule**
Data is now tracked in this open data repo since beginning of March: https://github.com/opencovid19-fr/data

which relies itself on this official government website: https://www.santepubliquefrance.fr/maladies-et-traumatismes/maladies-et-infections-respiratoires/infection-a-coronavirus/articles/infection-au-nouveau-coronavirus-sars-cov-2-covid-19-france-et-monde

- [x] Update some missing numbers for beginning of March #899
- [x] Bugfixes for #641, #816, #902 

Please note that: 
- I fixed missing values that were supposed to be constant from day to day here, I did not modify the aggregated `time_series` file on purpose to let you regenerate it.
- I did not find the number of recoveries in the reports,
- there is some double counting in the data you collected for [Overseas France](https://en.wikipedia.org/wiki/Overseas_departments_and_territories_of_France) (only 18 confimed cases since 11 of March, unfixed in the PR)

### CODE REVIEW

File: csse_covid_19_data/csse_covid_19_daily_reports/03-05-2020.csv [hunk 1]
```diff
@@ -17,7 +17,7 @@ Sichuan,Mainland China,2020-03-05T14:53:03,539,3,425,30.6171,102.7103
 ,Germany,2020-03-05T17:43:03,482,0,16,51.0000,9.0000
 Heilongjiang,Mainland China,2020-03-05T00:23:06,481,13,379,47.8620,127.7615
 Beijing,Mainland China,2020-03-04T23:23:01,418,8,297,40.1824,116.4142
-,France,2020-03-05T15:13:22,377,6,12,47.0000,2.0000
+,France,2020-03-05T15:13:22,423,5,12,47.0000,2.0000
```
Comment (mycaule) [inline csse_covid_19_data/csse_covid_19_daily_reports/03-05-2020.csv]: Source: https://www.santepubliquefrance.fr/content/download/235532/2526871

File: csse_covid_19_data/csse_covid_19_daily_reports/03-06-2020.csv [hunk 2]
```diff
@@ -12,7 +12,7 @@ Jiangxi,Mainland China,2020-03-06T14:23:04,935,1,909,27.6140,115.7221
 Shandong,Mainland China,2020-03-06T11:03:10,758,6,618,36.3427,118.1498
 Diamond Princess cruise ship,Others,2020-03-06T01:29:39,696,6,40,35.4437,139.6380
 ,Germany,2020-03-06T17:53:03,670,0,17,51.0000,9.0000
-,France,2020-03-06T20:13:13,653,9,12,47.0000,2.0000
+,France,2020-03-06T20:13:13,613,9,12,47.0000,2.0000
```
Comment (mycaule) [inline csse_covid_19_data/csse_covid_19_daily_reports/03-06-2020.csv]: Source: https://www.santepubliquefrance.fr/content/download/235535/2526923

File: csse_covid_19_data/csse_covid_19_daily_reports/03-09-2020.csv [hunk 3]
```diff
@@ -6,7 +6,7 @@ Hubei,Mainland China,2020-03-09T14:33:03,67743,3008,46488,30.9756,112.2707
 Guangdong,Mainland China,2020-03-09T01:43:01,1352,8,1260,23.3417,113.4244
 Henan,Mainland China,2020-03-08T05:03:02,1272,22,1247,33.8820,113.6140
 Zhejiang,Mainland China,2020-03-09T09:43:03,1215,1,1176,29.1832,120.0934
-,France,2020-03-09T00:43:03,1209,19,12,47.0000,2.0000
+,France,2020-03-09T00:43:03,1412,30,12,47.0000,2.0000
```
Comment (mycaule) [inline csse_covid_19_data/csse_covid_19_daily_reports/03-09-2020.csv]: Source: https://web.archive.org/web/20200309183257/https://www.santepubliquefrance.fr/maladies-et-traumatismes/maladies-et-infections-respiratoires/infection-a-coronavirus/articles/infection-au-nouveau-coronavirus-sars-cov-2-covid-19-france-et-monde

File: csse_covid_19_data/csse_covid_19_daily_reports/03-12-2020.csv [hunk 4]
```diff
@@ -3,7 +3,7 @@ Hubei,China,2020-03-12T09:53:06,67781,3056,50318,30.9756,112.2707
 ,Italy,2020-03-11T21:33:02,12462,827,1045,43.0000,12.0000
 ,Iran,2020-03-12T11:13:27,10075,429,2959,32.0000,53.0000
 ,"Korea, South",2020-03-12T05:13:02,7869,66,333,36.0000,128.0000
-France,France,2020-03-11T22:53:03,2281,48,12,46.2276,2.2137
+France,France,2020-03-11T22:53:03,2876,61,12,46.2276,2.2137
```
Comment (mycaule) [inline csse_covid_19_data/csse_covid_19_daily_reports/03-12-2020.csv]: Source: https://web.archive.org/web/20200312205456/https://www.santepubliquefrance.fr/maladies-et-traumatismes/maladies-et-infections-respiratoires/infection-a-coronavirus/articles/infection-au-nouveau-coronavirus-sars-cov-2-covid-19-france-et-monde

**Comment by JiPiBi** [comment]
The France recovered were updated globally  yesterday , you can find some daily infos in regional ARS sites or in local sites (what I do for my region) 
I fear as well that the number of cases is now not very significant , because the doctors say  that they confine at home moderately sick people (it your own doctor that takes the decision ) and I am not sure that the agregation is really made , and when  a person is confined probably his family is affected , but how these values are recorded ?
More interesting is to follow the number of person in ICU and hospitalized , like for recovered ,locally you can get some informations 
I suppose that with a person by region it should be easy to record these values (but the hour of the information could be another issue : at what time are the national values agregated for the daily report , must be tested on several days )

**Pull request closed without merging**